generators/pulsar/templates/gradle/pulsar.gradle.ejs

```diff
@@ -0,0 +1,9 @@
+dependencies {
+    implementation "org.springframework.cloud:spring-cloud-stream"
+    implementation ("org.springframework.pulsar:spring-pulsar-spring-cloud-stream-binder:${springPulsarVersion}") {
+        exclude group: 'org.springframework.boot', module: 'spring-boot-starter'
```

Comment:
Conflict with sb 3.0.2?
Please add a description.

Reply:
Yes. It pulls SB 3.0.5 and undertow autoconfig is broken. Between patch versions...

Reply:
`3.0.3` moved to UT`2.3.4.Final` as per https://github.com/spring-projects/spring-boot/issues/34273

The UT dependency has been steady in 3.0.3+
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/3.0.(3|4|5)

- `3.0.3 -> 2.3.4.Final`
- `3.0.4 -> 2.3.4.Final`
- `3.0.5 -> 2.3.4.Final`


The change in SB `3.0.3+` does require this change in Undertow in https://github.com/undertow-io/undertow/commit/e2a1f3ad39d96312f3befbe302629f4a4c2b5cdb

Aka the newly added `DECODE_SLASH`.

@cbornet can you verify what version of UT is being used? I wonder if its older (w/o the newly added field).

Reply:
If Jhipster is on `3.0.2` I could see how this would be an issue though.

Basically if you go from SB `3.0.2` to SB `3.0.3+` you have to upgrade Undertow as well. 😿 .

Reply:
Except there was a change in SB to use that field. So if the undertow being pulled in the old one...

Reply:
> So if the undertow being pulled in the old one...

Agreed - w/o the update to undertow we get the exception.